test/v0-open-load.js

```diff
@@ -123,7 +124,62 @@ Object.keys(testAPIs).forEach(API => {
 
       it('allows migrated key to write', async () => {
         const hash = await db.add({ thing: 'new addition' })
-        const newEntries = db.all.filter(e => e.v === 1)
+        const newEntries = db.all.filter(e => e.v > 0)
+        assert.equal(newEntries.length, 1)
+        assert.strictEqual(newEntries[0].hash, hash)
+      })
+    })
+
+    describe('Open & Load - V1 entries', function () {
+      const dbPath2 = './orbitdb/tests/v1'
+      const dbv1Fix = './test/fixtures/v1/QmZrWipUpBNx5VjBTESCeJBQuj4rWahZMz8CV8hBjdJAec/cache'
+      const v1Address = '/orbitdb/zdpuAqpKBwd7ojM77o3rRVKA1PAEQBnWoRASY3ugJ7zqnM6z7/v1-entries'
+      before(async () => {
+        await fs.copy(dbv1Fix, dbPath2)
+        db = await orbitdb.open(v1Address, { directory: dbPath2 })
+        await db.load()
+      })
+
+      beforeEach(async () => {
+        if (process.platform === 'win32') {
+          // for some reason Windows does not load the database correctly at the first time.
+          // this is not a good solution but... it works.
+          await db.load()
+        }
+      })
+
+      after(async () => {
+        rmrf.sync(dbPath2)
+        if (db)
+          await db.close()
+      })
+
+      it('open v1 orbitdb address', async () => {
+        assert.notEqual(db, null)
+      })
+
+      it('database has the correct v1 address', async () => {
+        assert.equal(db.address.toString().indexOf('/orbitdb'), 0)
+        assert.equal(db.address.toString().indexOf('zd'), 9)
+        assert.equal(db.address.toString().indexOf('v1-entries'), 59)
+      })
+
+      it('has the correct type', async () => {
+        assert.equal(db.type, 'feed')
+      })
+
+      it('database has the correct access-controller', async () => {
+        assert.equal(db.access.type, 'ipfs')
+        assert.strictEqual(db.access.write[0], '*')
+      })
+
+      it('load v1 orbitdb address', async () => {
+        assert.equal(db.all.length, 100)
+      })
+
+      it('allows adding new entry', async () => {
+        const hash = await db.add('new entry')
+        const newEntries = db.all.filter(e => e.v > 1)
```

Comment:
This is great 👍 Would be good to add one more test after this where we test that the db can be opened after adding the new entry to it.